Makefile

```diff
@@ -27,3 +27,11 @@ backstopjs-reference:
 
 backstopjs-test:
 	docker-compose exec backstopjs backstop test
+
+setup:
+	cp docker-compose.override.yml.dist docker-compose.override.yml
+	cp web/sites/default/example.settings.local.php web/sites/default/settings.local.php
```

Comment:
Would you mind if we add `default` as a Makefile argument? It would help when we have to use this script in multisite.

If would became something like this:
```
cp web/sites/$SITE/example.settings.local.php web/sites/$SITE/settings.local.php
```